lib/validator/validator.go

```diff
@@ -34,7 +34,7 @@ var validate *validator.Validate
 var (
 	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]{1,128}$")
 	interfaceRegex     = regexp.MustCompile("^[a-zA-Z0-9_-]{1,15}$")
-	labelRegex         = regexp.MustCompile("^([a-zA-Z0-9.-]{0,253}/)?[a-zA-Z0-9]([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?$")
+	labelRegex         = regexp.MustCompile(`^([a-zA-Z0-9\_\.\-\/]{0,252}[a-zA-Z0-9])?$`)
```

Comment:
> yeah the only one that could have been without escape was / but I did it to be consistent.

Interesting, what is the behavior of `_`, for example, when not escaped?

Also, no need to be consistent - it's more confusing the unnecessarily escape something (might come back later and be confused by it)